manifests/certonly.pp

```diff
@@ -182,11 +186,18 @@
     $cron_script_ensure = $ensure ? { 'present' => 'file', default => 'absent' }
     $cron_ensure = $ensure
 
-    if $suppress_cron_output {
-      $croncommand = "${maincommand} > /dev/null 2>&1"
-    } else {
-      $croncommand = $maincommand
+    case $cron_output {
```

Comment:
I would use selector syntax here since it's more clean.

Reply:
Made this change.